ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/node_type_keys.py

```diff
@@ -42,16 +38,10 @@
             },
             "underlay_router": False,
             "uplink_type": "port-channel",
+            "cloudvision_tags_topology_type": "leaf",
         },
-        {
-            "key": "super_spine",
-            "type": "super-spine",
-        },
-        {
-            "key": "overlay_controller",
-            "type": "overlay-controller",
-            "default_evpn_role": "server",
-        },
+        {"key": "super_spine", "type": "super-spine", "cloudvision_tags_topology_type": "core"},
```

Comment:
please add comma to the end of the last value so `black` will unfold these

Reply:
Added the ',' so these lines are unfolded.